src/scrappers/source/proCyclingStats/raceStages.js

```
@@ -20,109 +19,178 @@ import { config } from "@scrappers/html/config-puppeteer";
+          break;
+
+        case "parcoursType":
+          const span = td.querySelector("span");
+          value = span
+            ? Array.from(span.classList).find((cls) => /^p\d+$/.test(cls))
+            : null;
+          details[key] = value;
+          break;
+
+        case "stage":
+          const anchor = td.querySelector("a");
```

Biome (2.4.10)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)